### ISSUE TITLE
volar.vue.findAllFileReferences modifies file

### ISSUE THREAD

**Issue by tkane2000** [vuejs/language-tools#4262]
## Description
In VSCode, when you run `volar.vue.findAllFileReferences` via `cmd+shift+P` or via a keyboard shortcut, the file in the current editor is modified.  There are no notable changes to the file and hitting undo twice bring it back to an unmodified state.

Everything else works as expected. 

## Expected behaviour
The file is not modified when you run `volar.vue.findAllFileReferences`.

## Environment
OS: MacOS Sonoma, Version 14.2.1 (23C71)
VSCode: v1.88.0
Vue Extension: v1.8.27 <------

Additional information
Waiting on https://github.com/vuejs/language-tools/issues/4240 to upgrade to v2.  ...not sure if the issue is fixed there already.

**Comment by johnsoncodehk** [comment]
`volar.vue.findAllFileReferences` command is removed in v2, you should use `typescript.findAllFileReferences` (When Hybrid Mode enabled) or `vue.findAllFileReferences` (When Hybrid Mode disabled).

**Comment by johnsoncodehk** [comment]
Duplicate of #4171
